TrafficCapture/trafficCaptureDependencies/build.gradle

```diff
@@ -0,0 +1,107 @@
+plugins {
+  id "java-library"
```

Comment:
I'm not sure all that's required, but i think we want to declare this project as a java-platform not library

https://docs.gradle.org/current/userguide/java_platform_plugin.html#sec:java_platform_consumption

and use
```
api platform(project(":trafficCaptureDependencies"))
```

Reply:
Seems to work as is, lets follow up in a fresh PR if there are improvements that can be made